src/v0.15/guide/data-flows.md

```diff
@@ -79,7 +80,7 @@ to error as well.
 
 Here are some guidelines for working with data flows:
 
-* Consider the full arc of each request - how it will flow up to be fulfilled,
+* Consider the full arc of each request—how it will flow up to be fulfilled,
```

Comment:
The emdash is an improvement 👍. I will need to verify how it looks without spaces for padding.

Reply:
I thought em dashes are supposed to use without spaces around them but I could easily be incorrect :) I just looked again and it seems like it is a matter of preference so LMK if you'd prefer the version with spaces and I can update

## Before

<img width="1212" alt="Screen Shot 2019-05-10 at 2 44 02 AM" src="https://user-images.githubusercontent.com/602204/57518376-b611e900-72cd-11e9-9151-ac0166c04bf0.png">

## After (current)

<img width="1212" alt="Screen Shot 2019-05-10 at 2 44 47 AM" src="https://user-images.githubusercontent.com/602204/57518362-b1e5cb80-72cd-11e9-9999-bc3924e1e61c.png">

## After (with spaces)

<img width="1174" alt="Screen Shot 2019-05-10 at 2 49 32 AM" src="https://user-images.githubusercontent.com/602204/57518643-57993a80-72ce-11e9-9b58-48fde2ec9069.png">

Reply:
Thanks for following up and for the screenshots. After doing some research myself, it does seems that it's quite common to leave out the spaces, so let's go with your current change.